cinema/views.py

```diff
@@ -1 +1,65 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import Movie, Genre, Actor, MovieSession, CinemaHall
+from cinema.serializers import (
+    MovieSerializer,
+    MovieListSerializer,
+    MovieRetrieveSerializer,
+    GenreSerializer,
+    ActorSerializer,
+    MovieSessionSerializer,
+    CinemaHallSerializer,
+    MovieSessionListSerializer,
+    MovieSessionRetrieveSerializer,
+)
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieSerializer
+
+    def get_queryset(self):
+        queryset = Movie.objects.all()
```

Comment:
As mentioned above, you do not need to use `.all()` before `.prefetch_related()` and `.select_related()` methods. So you can remove `.all()` here.